src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import lotto.function.GenerateNumbers;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumbers {
+  private static final int LOTTO_NUMBER_TOTAL_COUNT = 6;
+  private static final String EXCEPTION_MESSAGE = "로또는 6자리 숫자로 생성해야 됩니다.";
+
+  private final List<LottoNumber> list;
```

Comment:
인스턴스 변수 네이밍을 값에 대한 타입으로 작성해주셨는데요!
만약 타입이 변경되게 된다면 네이밍을 수정해줘야 하는 이슈가 있을 것 같아요!
```suggestion
  private final List<LottoNumber> values;
```
다음과 같은 네이밍은 어떨까요? 😄 
(다른 클래스들도 확인 부탁드립니다 😄 )